apps/ledger-live-desktop/src/newArch/features/ModularDrawer/components/SelectAssetFlow.tsx

```diff
@@ -64,25 +42,22 @@ function SelectAssetFlow(props: SelectAccountAndCurrencyDrawerProps) {
           ff="Inter|Medium"
           color="palette.text.shade100"
           fontSize="24px"
-          style={{
-            textTransform: "uppercase",
-          }}
           data-testid="select-asset-drawer-title"
         >
           {t("drawers.selectCurrency.title")}
         </Text>
       </HeaderContainer>
       <SelectorContent>
         <SearchInputContainer>
-          <SearchInput
-            data-testid="select-asset-drawer-search-input"
-            value={searchValue}
-            onChange={setSearchValue}
+          <SearchItem
```

Comment:
It's fine to replace the search here so we see it, but the search should be on the SelectAsset component. Because it only shows durring the asset selection not the network selection